pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -296,6 +301,12 @@ func (c *Cluster) Start() error {
 		return errors.Wrap(err, "failed post reconcile of osd properties")
 	}
 
+	// TODO: update CephCluster status
+	// if a user initiates rotation but then decides partway through not to continue,
+	// we can have a non-homogeneous state where some osds are at gen A / cephversion A
+	// others at gen B / cephversion B. extreme examples could have all OSDs at diff values
+	// tracking every OSD seems unnecessary and bloated -- what about tracking min and max vals?
```

Comment:
Any thoughts on this @travisn ?

Reply:
As discussed in huddle, having this update only when all update is good for now. Corner cases are possible, but unlikely, and we might be able to help better indicate status for users later when key creation time is available.